import-export-cli/cmd/getKeys.go

```diff
@@ -48,22 +48,22 @@ var subscriptionThrottlingTier string
 var applicationThrottlingPolicy string
 var keyGenTokenEndpoint string
 
-var genKeyCmd = &cobra.Command{
-	Use:     genKeyCmdLiteral,
-	Short:   genKeyCmdShortDesc,
-	Long:    genKeyCmdLongDesc,
-	Example: genKeyCmdExamples,
+var getKeysCmd = &cobra.Command{
+	Use:     GetKeysCmdLiteral,
+	Short:   getKeysCmdShortDesc,
+	Long:    getKeysCmdLongDesc,
+	Example: getKeysCmdExamples,
 	Run: func(cmd *cobra.Command, args []string) {
 
-		utils.Logln(utils.LogPrefixInfo + genKeyCmdLiteral + " called")
+		utils.Logln(utils.LogPrefixInfo + GetKeysCmdLiteral + " called")
 		getKeys()
 	},
 }
 
 //Subscribe the given API or API Product to the default application and generate an access token
 func getKeys() {
```

Comment:
The logic of `getKeys()` is duplicated in both the `cmd` and `deprecated` package versions of this command. We need to eleimnate this duplication by moving this logic into the impl layer and reusing the same within `cmd` and `deprecated` packages.

Reply:
Fixed via https://github.com/wso2/product-apim-tooling/pull/515/commits/4db36fd299a47e5e82dbe448b075824be1244efa